src/components/index.js

```diff
@@ -0,0 +1,6 @@
+import { Navbar } from "./Navbar";
+import { Hero } from "./Hero";
+import { Categories } from "./Categories";
+import { Footer } from "./Footer";
+
+export { Navbar, Hero, Categories, Footer };
```

Comment:
Instead of importing and then exporting, you can directly export
`export { componentName } from "path/to/file`

Reply:
ah.. yes  will make that change